user-journey.adoc

```diff
@@ -1,62 +1,91 @@
-= User Journey for Istio/Jaeger
+= User Journey - Installing Istio + Fabric8 Launcher
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
+This document describes how to use the Istio installer to install the following components:
```

Comment:
I don't believe we can simplify down this far. There's a lot of operating system setup hapening here. It's not just "using an installer" that's one step in a fairly complex process.

Reply:
I was actually planning to try to track down all the configuration that the installer is performing under the covers and list it (so that users know they don't need to do it).  But that was going to be in the next version of the doc, because that's going to require a bit of research.